File: fix_stack_dynamic.java
public class FixedStackDynamicExample {

    public static void main(String[] args) {
        // Fixed Stack-Dynamic Array
        int[] numbers = new int[5];  // size is fixed at runtime

        // Assign values
        for (int i = 0; i < numbers.length; i++) {
            numbers[i] = i + 10;
        }

        // Display the array
        System.out.println("Fixed Stack-Dynamic Array:");
        for (int num : numbers) {
            System.out.print(num + " ");
        }
    }
}

File: fix_stack_dynamic.py
def fixed_stack_dynamic():
    # Fixed size list created inside function (stack memory)
    arr = [0] * 5  # fixed size of 5 elements

    # Assign values
    for i in range(len(arr)):
        arr[i] = i + 1

    # Display the list
    print("Fixed Stack-Dynamic Array:")
    print(arr)

fixed_stack_dynamic()
